### DISCUSSION TITLE
Is 1.6.2 officially out?

### DISCUSSION THREAD

**Discussion by giveitatry168** [edenhill/librdkafka#3628] [Q&A] [answered]
The following commit bumps the version.
     https://github.com/edenhill/librdkafka/commit/4483be32ba833e0a2a9129e2510bd63a40655b43

but couldn't find 1.6.2 under https://github.com/edenhill/librdkafka/releases

**Comment by edenhill** [comment, marked as answer]
The final release has not been tagged yet, but will be soon.

This will be a tag-only release, there will not be any nuget packages or daughter client releases for this one.
